portfolio/src/main/webapp/script.js

```diff
@@ -12,16 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
+// Adds "Hello Andrew" to page 
+async function getGreetingAsyncAwait() {
+  const response = await fetch('/data');
+  const greeting = await response.text();
+  document.getElementById('greeting-container').innerText = greeting;
```

Comment:
duplicate logic?

Reply:
Is this in regards to the two 'await' calls? - using async versus arrow function promise chain?
I saw both in the walkthrough example

Reply:
No, github was showing "document.getElementById('greeting-container').innerText = greeting;" and the line below it was the same logic but in two lines.